ecosystem/sep-0010.md

```diff
@@ -139,7 +139,75 @@ operation of the challenge is the **Home Domain**, and that the web auth domain
 GET <WEB_AUTH_ENDPOINT>
 ```
 
-Request Parameters:
+##### Request Header:
+
+Optionally, the server may forbid unauthorized calls to `GET <WEB_AUTH_ENDPOINT>`. This could be done to protect an
+endpoint against bad actors or limit access only to allowed applications.
+
+In that case, the client must add an `Authorization` header, and the server should respond to all requests with missing
+header with an error (see below). Server may choose to only accept calls from trusted applications, responding with an
+error to all other requests (see below).
+
+Authorized `WEB_AUTH_ENDPOINT` accepts `Authorization Bearer: <token>`, where token is [JSON Web Token](https://jwt.io/)
+authorizing the request.
+
+Client must specify the following claims:
+
+- `iat` (the time at which the JWT was issued
+  [RFC7519, Section 4.1.6](https://tools.ietf.org/html/rfc7519#section-4.1.6)) — current timestamp (`1530644093`)
+- `exp` (the expiration time on or after which the JWT must not be accepted for processing,
+  [RFC7519, Section 4.1.4](https://tools.ietf.org/html/rfc7519#section-4.1.4)) — a server can pick its own expiration
+  period for the token (`1530730493`)
```

Comment:
This JWT is generated by the client, right? So when you say "a server can pick its own expiration", are you referring to the client? We may want to disambiguate that.

Reply:
Yes, I just copy-pasted it 😅  I'll just remove this part